shopinvader/models/product_product.py

```diff
@@ -1,5 +1,7 @@
 # Copyright 2017 Akretion (http://www.akretion.com).
 # @author Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2019 Camptocamp (http://www.camptocamp.com).
+# @author Simone Orsi <simone.orsi@camptocamp.com>
```

Comment:
@simahawk IMO we must avoid to add copyright for each develeopper into each file we modify (otherwise I'll have to add one 2 lines for acsone into all the shopinvader files ;-) author and contributors are respectively specified into the manifest and the authors.rst file